google/users.go

```diff
@@ -414,45 +472,28 @@ func updateLocations(newArea string, oldLocations interface{}) ([]admin.UserLoca
 	return locations, nil
 }
 
-func updatePhones(newPhone string, oldPhones interface{}) ([]admin.UserPhone, error) {
-	phones := []admin.UserPhone{{Type: "work", Value: newPhone}}
+func attributesToUserPhones(phones map[string]string) ([]admin.UserPhone, error) {
+	userPhones := []admin.UserPhone{}
 
-	if oldPhones == nil {
-		return phones, nil
-	}
-
-	interfaces, ok := oldPhones.([]interface{})
-	if !ok {
-		return nil, errors.New("no slice in Google API Phones")
-	}
-
-	for i := range interfaces {
-		phoneMap, ok := interfaces[i].(map[string]interface{})
-		if !ok {
-			return nil, errors.New("unexpected data in Google API phone list")
-		}
-
-		thisType, ok := phoneMap["type"].(string)
-		if !ok {
-			return nil, errors.New("unexpected data in Google API phone list entry")
+	for key, val := range phones {
+		split := strings.Split(key, ",")
```

Comment:
Just wondering if the `","` here should be `delim`

Reply:
Yeah, and the `beforeDelim` function too. Thanks